NEWS.md

```diff
@@ -1,5 +1,9 @@
 # r2dii.analysis (development version)
 
+* `target_market_share()` now groups by region. This fixes a bug by which
+`technology_share` unexpectedly increased monotonically with time
+(@Antoine-Lalechere #315).
```

Comment:
This is my interpretation of the problem we fixed but I am likely wrong, as the patch that @Antoine-Lalechere provided didn't pass the test that meets this requirement. 

@Antoine-Lalechere and @jdhoffa please help me develop a more precise test.